openvpn-install.sh

```diff
@@ -1286,6 +1286,33 @@ function removeOpenVPN() {
 	fi
 }
 
+function listCerts () {
+# Original Script from PiVPN: list clients script
+# Modified Script to add Certificate expiration Date -- psgoundar  
+INDEX="/etc/openvpn/easy-rsa/pki/index.txt"
+printf "\\n"
+if [ ! -f "${INDEX}" ]; then
+        echo "The file: $INDEX was not found!"
+        exit 1
+fi
+printf "\\e[1m::: Certificate Status List :::\\e[0m\\n"
+printf "\\e[4mStatus\\e[0m ::  \\e[4mName\\e[0m\\e[0m        ::  \\e[4mExpiration \\e[0m\\n" 
+while read -r line || [ -n "$line" ]; do
+    STATUS=$(echo "$line" | awk '{print $1}')
+    NAME=$(echo "$line" | awk '{print $5}' | awk -FCN= '{print $2}')
```

Comment:
```suggestion
    NAME=$(echo "$line" | cut -d '=' -f2)
```
This doesn't work when clients are revoked, I changed it.